protocols/ML01.md

```diff
@@ -0,0 +1,315 @@
+# ML01: Machine learning basic specification
+**THIS IS A DRAFT SPECFICATION AND IS NOT FINAL**
+
+| name                       | ML01               |
+|----------------------------|--------------------|
+| version                    | 0.1.0              |
+| status                     | proposal           |
+| author                     | Ryan Wilson (rw86) |
+| serving component(s)       | Machine Learning   |
+| consuming component(s)     | HCI                |
```

Comment:
But... but.... we voted against this... like the whole class voted??? :'(

Reply:
My fault - see above on how this would be changed